### PR TITLE
sql/expression: new date function

### PR DESCRIPTION
**Pull request by mcuadros**
Signed-off-by: Máximo Cuadros <mcuadros@gmail.com>

Fix: https://github.com/src-d/go-mysql-server/issues/653

### CODE REVIEW

File: server/handler.go [hunk 1]
```diff
@@ -77,6 +77,10 @@ func (h *Handler) ComQuery(
 ) (err error) {
 	ctx := h.sm.NewContextWithQuery(c, query)
 
+	if strings.ToLower(query) == "show schemas" {
```
Comment (kuba--) [inline server/handler.go]: not related to this PR , but whatever if we can handle `show schemas`, but afaik it's not merged into vitess, yet.

Comment (mcuadros) [reply server/handler.go]: Was upload by error sorry

**Comment by mcuadros** [comment]
done

**Comment by erizocosmico** [comment]
@ajnavarro shall we merge this?

**Pull request merged**